src/core/node/node.builder.js

```diff
@@ -13,16 +13,17 @@ export default function nodeBuilder(columnMap, groupBy, valueFactory, level = 0)
 
 	const column = columnMap[groupKey];
 	const getValue = valueFactory(column);
-	return rows => {
+	return (rows, getIdentity = (i) => i) => {
```

Comment:
the name should be getRowIndex, identity means return passed arg as  it is :)
https://en.wikipedia.org/wiki/Identity_function